Fantasy Battle Game/Assets/Scripts/Champions/Scripts/ChampionsManager.cs

```diff
@@ -3,12 +3,22 @@
 using System.Linq;
 using UnityEditor;
 using UnityEngine;
+using UnityEngine.Tilemaps;
 
 public class ChampionsManager : MonoBehaviour
 {
     public static ChampionsManager Instance;
```

Comment:
This thing should not be public. It is field, not property. It is just formality, yet really important one. Once we will try to change behavior, it may strike us back. We should consider it an issue.